vsphere_storage_plugin/cidata.py

```diff
@@ -91,3 +91,32 @@ def delete(rawvolume_client, **kwargs):
     rawvolume_client.delete_file(datacenter_id=datacenter_id,
                                  datacenter_name=datacenter_name,
                                  datastorepath=storage_path)
+
+
+@op
+@with_rawvolume_client
+def upload_iso(rawvolume_client,
+               datacenter_name,
+               allowed_datastores,
+               allowed_datastore_ids,
+               volume_prefix,
+               iso_file_path,
+               **_):
+    is_node_deprecated(ctx.node.type)
```

Comment:
We can probably replace with with the deprecation_warning decorator.

Reply:
will create another task to go over all the plugin and change the logic there, maybe make check deprecation part of op decorator